cloud-consumption/src/main/java/com/sequenceiq/consumption/flow/consumption/storage/event/StorageConsumptionCollectionEvent.java

```diff
@@ -11,7 +14,13 @@ public StorageConsumptionCollectionEvent(String selector, Long resourceId, Strin
         super(selector, resourceId, resourceCrn);
```

Comment:
shouldn't the same be added to this constructor as well?

Reply:
nope